src/__tests__/sentry-test.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+import * as Sentry from '@sentry/react-native';
+
+describe('sentry', () => {
+  describe('is usable without initialization', () => {
+    const expectSentryUninitialized = () => {
+      // Hub#getClient() is documented as possibly returning undefined, but the
+      // significance of `undefined` is not. In practice, it appears to be
+      // `undefined` exactly when `Sentry.init()` has not yet been called.
+      expect(Sentry.getCurrentHub().getClient()).toBeUndefined();
+    };
+
+    test('breadcrumbs', () => {
+      expectSentryUninitialized();
+
+      Sentry.addBreadcrumb({
+        message: 'test message; should not result in a real breadcrumb',
+        level: Sentry.Severity.Debug,
+      });
+    });
```

Comment:
That's fair.

The second one should probably stay essentially as is, though, to prevent some hypothetical future reader of Sentry event logs dismissing `Jest test error` as harmless expected noise. I've tagged it with a comment.

Reply:
Ah, I see!
```
+      // The text here is intended to prevent some hypothetical future reader of
+      // Sentry event logs dismissing the error as harmless expected noise, in
+      // case Sentry is somehow actually initialized at this point.
+      const err = new Error('Jest test error; should not result in a Sentry event
');
```

Makes sense; and thanks for the comment explaining it.